tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "importHelpers": true,
     "esModuleInterop": true,
     "outDir": "dist",
-    "declaration": false,
+    "declaration": true,
```

Comment:
```suggestion

```

[the default is `true`](https://www.typescriptlang.org/tsconfig#declaration)

Reply:
Typescript docs is wrong. You can try delete this line then no `.d.ts` produced.

Reply:
Thanks for pointing this out, made a PR upstream to fix those docs: https://github.com/microsoft/TypeScript-Website/pull/1092